crates/store/src/lib.rs

```diff
@@ -30,6 +46,17 @@ pub enum CreateFromReaderError<E> {
     Create(CreateError<E>),
 }
 
+pub trait New<K>
```

Comment:
This seems like it should not be necessary and it is not clear what it's for. At the very least this needs documentation.

Reply:
This basically just exists to bind Filesystem and Memory together. I can add a comment assuming we want to keep this around.

Reply:
removed this